data/amplio2/units.spec

```diff
@@ -52,7 +52,7 @@ tiles = { "row", "column", "tag"
   0,  1, "u.howitzer_Idle"      ; [Nemo]
   0,  2, "u.battleship_Idle"    ; [Nemo]
   0,  3, "u.bomber_Idle"        ; [GB]
-  0,  4, "u.cannon_Idle"        ; [CT]
+  0,  4, "u.early_cannon_Idle"  ; [CT]
```

Comment:
Changes like this are a bit more complicated. The main units gfx file should have all the standard units (those that oure included rulesets use). "extra_units" are just for more custom units.

So, if you want sprite from extra_units to become the official cannon, you need to switch the sprites in the .png -files instead.

Reply:
I swapped some images in the trident set, and restored the original Amplio2 set.